api/v1alpha2/linodecluster_types.go

```diff
@@ -43,6 +43,11 @@ type LinodeClusterSpec struct {
 	// +optional
 	Network NetworkSpec `json:"network"`
 
+	// UseVlan provisions a cluster that uses VLANs instead of VPCs. IPAM is managed internally.
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="Value is immutable"
+	// +optional
+	UseVlan bool `json:"useVlan"`
```

Comment:
Not sure if we want to keep this here or move it to the NetworkSpec

Reply:
agreed I think this makes sense in the NetworkSpec